core/src/main/scala/org/broadinstitute/workbench/hamm/db/JobTableQueries.scala

```diff
@@ -0,0 +1,111 @@
+package org.broadinstitute.workbench.hamm.db
```

Comment:
Scalike queries for the Job table: 
- insert job
- get job
- get job cost
- get job's workflow's collection id   (<--- why I added a foreign key on the workflow table)